android/modules/ui/src/java/ti/modules/titanium/ui/widget/searchview/TiUISearchView.java

```diff
@@ -47,19 +49,21 @@ public void processProperties(KrollDict props) {
 
 		// Check if the hint text is specified when the view is created.
 		if (props.containsKey(TiC.PROPERTY_HINT_TEXT)) {
-			searchView.setQueryHint(props.getString(TiC.PROPERTY_HINT_TEXT));			
-		} 
+			searchView.setQueryHint(props.getString(TiC.PROPERTY_HINT_TEXT));
+		}
+		changeEventEnabled = false;
```

Comment:
Hi @garymathews, if I put line 54 ( `changeEventEnabled = false;`) inside the `if` , I would cause this bad behaviour: 
on JS side if you try to run this code and then with softKeyboard manually change the text of the SearchView you will not see the alert (which is not the standard behaviour), this is because in native side the changeEventEnabled var is set to `false` and will never change to `true` until the `value` prop is set on method processProperties. That's why I left line 54 outside `if` statement.
`var win = Ti.UI.createWindow();
 
var searchView = Ti.UI.Android.createSearchView({
hintText:"abcde" 
});
 
searchView.addEventListener("change",function(){
    alert("Fired Change Event!");
});
 
win.add(searchView);
win.open();`